### ISSUE TITLE
Cloud testing: Credentials and Workflows

### ISSUE THREAD

**Issue by zeier** [agera-edc/DataSpaceConnector#192]
Feature #285
Follow-up story for #146

**Comment by chrislomonico** [comment]
Blocked until #146 is unblocked

**Comment by zeier** [comment]
Feature is deprioritized and won't be part of the scope until end of August
